Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM golang:alpine3.19@sha256:0466223b8544fb7d4ff04748acc4d75a608234bf4e79563bff208d2060c0dd79
```

Comment:
do you know if dependabot automatically bumps these? I think it's easier to use the `golang:goversion` images to see what go version are we using (I don't particulary care about the OS version, we're not installing anything anyway)